pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -950,9 +960,23 @@ func ValidateVirtualMachineInstanceMandatoryFields(field *k8sfield.Path, spec *v
 	return causes
 }
 
-func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, metadata *metav1.ObjectMeta, config *virtconfig.ClusterConfig) []metav1.StatusCause {
+func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, metadata *metav1.ObjectMeta, config *virtconfig.ClusterConfig, accountName string) []metav1.StatusCause {
+
 	var causes []metav1.StatusCause
 	annotations := metadata.Annotations
+	labels := metadata.Labels
+	// Validate kubevirt.io labels presence. Restricted labels allowed
+	// to be created only by known service accounts
+	allowed := getAllowedServiceAccounts()
+	if _, ok := allowed[accountName]; !ok {
+		if len(filterKubevirtLabels(labels)) > 0 {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueNotSupported,
+				Message: "creation of the following reserved kubevirt.io/ labels on a VMI object is prohibited",
```

Comment:
Are you sure that this error message actually contains the label?

Reply:
@rmohr I didn't include the concrete label intentionally since it would require to iterate all the restricted labels and check which added which deleted and which modified with content. I prefer to defer this enhancement of information to a separate PR, and add these restrictions to our documentation.

Reply:
Well, the issue is  just that the test says that the "following" labels starting with "kubevirt.io/" are prohibited. However they are never mentioned then. :)

Reply:
@rmohr You are correct! I missed that one.

Reply:
Let's ensure we do a follow up PR.

Reply:
@rmohr Sure